src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.head_a {
+  height: 60px;
+}
+
+.header img {
+  margin: 10px 0;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav_item {
+  display: inline-block;
+  text-transform: uppercase;
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: Roboto, sans-serif;
```

Comment:
You should use a fallback font in case 'Roboto' is not available on the user's computer. The fallback font should be a similar style to your chosen font. For example, you could use 'Arial' as a fallback to 'Roboto'.